Sources/MapboxNavigation/NavigationCameraStateTransition.swift

```diff
@@ -118,6 +117,11 @@ public class NavigationCameraStateTransition: CameraStateTransition {
               let padding = cameraOptions.padding else { return }
         
         let duration = 1.0
+        let centerCoordinatePixelThreshold = 2.0
+        let pitchThreshold = 1.0
+        let bearingThreshold = 1.0
```

Comment:
Thanks for factoring out these thresholds. It would be a bit clearer to name the variables more descriptively as “minimum”s and give them more specific types like `CLLocationDirection`.

Reply:
Makes sense, changed.